File: apps/android/app/src/main/kotlin/com/nogeass/passalarm/domain/model/ScheduledToken.kt
package com.nogeass.passalarm.domain.model

data class ScheduledToken(
    val id: Long = 0,
    val planId: Long = 0,
    val date: String = "",
    val fireAtEpoch: Long = 0,
    val osIdentifier: Int = 0,
    val status: TokenStatus = TokenStatus.PENDING,
    val createdAt: Long = System.currentTimeMillis(),
    val updatedAt: Long = System.currentTimeMillis()
)

File: apps/ios/PassAlarm/Domain/Entity/ScheduledToken.swift
import Foundation

struct ScheduledToken: Identifiable, Equatable, Sendable, Codable {
    var id: UUID
    var planId: UUID
    var date: String // "YYYY-MM-DD"
    var fireAtEpoch: TimeInterval
    var osIdentifier: String
    var status: TokenStatus
    var createdAt: Date
    var updatedAt: Date
}
